gofer.spec

```diff
@@ -208,6 +212,7 @@ Requires: python-qpid-proton >= 0.9-5
 Provides the gofer qpid proton messaging adapter package.
 
 %files -n python-%{name}-proton
+%defattr(-,root,root,-)
```

Comment:
Some %files blocks have this and some don't. Is it need for all of them, or possibly for none of them?